Source/WebGPU/Configurations/wgslfuzz.xcconfig

```diff
@@ -25,4 +25,8 @@ PRIVATE_HEADERS_FOLDER_PATH = $(WGSL_INSTALL_PATH_PREFIX)$(WK_LIBRARY_HEADERS_FO
 PUBLIC_HEADERS_FOLDER_PATH = $(WGSL_INSTALL_PATH_PREFIX)$(WK_LIBRARY_HEADERS_FOLDER_PATH);
 HEADER_SEARCH_PATHS = "$(BUILT_PRODUCTS_DIR)$(WK_LIBRARY_HEADERS_FOLDER_PATH)" $(inherited);
 SYSTEM_HEADER_SEARCH_PATHS = $(SDK_DIR)$(WK_ALTERNATE_WEBKIT_SDK_PATH)$(WK_LIBRARY_HEADERS_FOLDER_PATH) $(inherited);
+
+SKIP_INSTALL = $(WK_NOT_$(ENABLE_LIBFUZZER));
+INSTALL_PATH = $(WEBKIT_FUZZERS_DIR)
```

Comment:
I might prefer that we install this to the WebGPU.framework, if the location is flexible. No need to start installing to a  brand new directory. Here's what we do in WebKitTestRunner: https://github.com/WebKit/WebKit/blob/fe663b06e8da7388f053104ca13de9ca48730212/Tools/WebKitTestRunner/Configurations/BaseTarget.xcconfig#L24-L27

Reply:
I understand you want to put the binary with the associated framework as a resource, but doing this makes it difficult to see a list of all the WebKit-project-related fuzzers in one place.

If we did this consistently, we'd put all of the WebCore fuzzers in `WebCore.framework/Resources`, but then where would the `libANGLE-dynamic` and `libwebrtc` fuzzers go since they are dylibs?

I guess I'd rather see them all go to one place first.

> No need to start installing to a brand new directory.

One of the main points of this PR is to install the fuzzers to a specific, well-defined location.